src/main/java/org/jboss/xavier/utils/Utils.java

```diff
@@ -0,0 +1,16 @@
+package org.jboss.xavier.utils;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class Utils {
+
+    public Date getUnixEpochDate() {
+        return new Date(0L);
+    }
```

Comment:
It seems like a duplicate of the same method in `DateUtils`

Reply:
you are right, this is definitely a duplicate code. I've fixed it